decorator.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'nameable'
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
```

Comment:
- [ ] Your code fails. As the previous reviewer mentioned, kindly add getter and setter (`attr_accessor`) for the `:nameable` variable.

  ```
  class Decorator < Nameable
    attr_accessor :nameable
  
    def initialize(nameable)
      super()
      @nameable = nameable
    end
    ...
  ```

  ![image](https://github.com/yusufyusie/School-Library/assets/36542986/4b504ac7-8be7-4306-96e2-f857dce44772)